### ISSUE TITLE
Disable Web RTC Ip address leakage in Firefox

### ISSUE THREAD

**Issue by petrpatek** [apify/fingerprint-suite#124]
**Describe the feature**
In its current state, Firefox leaks the Ip address in Web RTC.

![](https://cdn.discordapp.com/attachments/1059916802446073957/1062402436960096336/pixelscan.net-pixelscan.net1-2.png)

**Motivation**
This could make the firefox browser even stealthier. 

**Constraints**
I did some digging, and it seems that the only way to prevent this is to disable `media.peerconnection.enabled` in "about:config". The only way I found to do this programmatically is to change or create a user.js configuration file for firefox. I am not sure if we can do this on the fingerprint-suite level.

Resources: http://kb.mozillazine.org/User.js_file, http://kb.mozillazine.org/Profile_folder

**Comment by metalwarrior665** [comment]
Comments from Discord user, to be tested
```
I want to add one thing disabling WebRtc would again trigger captchas. So we have to find a way to emulate WebRtc. Then it will be a real stealth.
```

**Comment by barjin** [comment]
@metalwarrior665 Running our current Cloudflare protection tests with the linked PR (#141) doesn't show any difference in the number of passed tests nor the mean time to pass. I'll get in touch with the Discord user and ask him if he has anything specific in mind, otherwise, this is a go from me.

**Comment by wilsonssss** [comment]
<img width="934" alt="image" src="https://github.com/apify/fingerprint-suite/assets/4098951/35352521-07db-40ff-ac1c-0b766bf840c3">

The same situation, some body help?

**Comment by barjin** [comment]
Hello, @wilsonssss, as you can see, this issue has been closed by #141 , which we have since merged and released. 

That PR added a new feature to `fingerprint-suite` that allows you to use the `mockWebRTC: true` option like this:

```typescript
const context = await newInjectedContext(
    browser,
    {
        fingerprintOptions: {
            mockWebRTC: true,
        },
    },
);
```

...to disable WebRTC in your injected browser. 

Thank you for your interest in the `fingerprint-suite` project!
